MAUI/Scheduler/header.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Header in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about to customize header height, date format and appearance of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more. 
```

Comment:
Learn here all about how to customize header height, date format, and appearance of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more.